lib/stellar/operation.rb

```diff
@@ -122,7 +122,14 @@ def self.create_account(attributes={})
     # @return [Stellar::Operation] the built operation, containing a
     #                              Stellar::ChangeTrustOp body
     def self.change_trust(attributes={})
-      line  = Asset.send(*attributes[:line])
+      line = attributes[:line]
+      if !line.is_a?(Asset)
+        if !Asset::TYPES.include?(line[0])
+          fail ArgumentError, "must be one of #{Asset::TYPES}"
```

Comment:
We tend to use `raise` in this codebase instead of `fail`.   They're synonymous, but IMO consistency is nice.